src/snowflake/snowpark/dataframe.py

```diff
@@ -1976,6 +1978,32 @@ def filter(
             _ast_stmt=stmt,
         )
 
+    def _filter_group_by_rows(
```

Comment:
I assume you will need to change all the corresponding code in sas to make usage of this new api?

Reply:
done